api/core/Directus/Slim/ResponseCacheMiddleware.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Directus\Slim;
+
+use Directus\Bootstrap;
+use Directus\Cache\Response;
+use Slim\Middleware;
+
+class ResponseCacheMiddleware extends Middleware
+{
+    public function call()
+    {
+        /** @var Response $cache */
+        $cache = Bootstrap::get('responseCache');
+
+        if($_SERVER['REQUEST_METHOD'] == 'GET') {
+            $path = parse_url($_SERVER["REQUEST_URI"], PHP_URL_PATH);
+
+            $parameters = $_GET;
+            ksort($parameters);
+
+            $key = md5($path.'?'.http_build_query($parameters));
+        } else {
+            $key = null;
+        }
+
+        if($key && $cachedResponse = $cache->get($key)) {
```

Comment:
Can the cache be disable? example on development environment or simple because we want it disable incase we are using a different solution?